django_admin_geomap/__init__.py

```diff
@@ -97,13 +98,14 @@ def changelist_view(self, request, extra_context=None):
         # Obtain original response from Django
         response = super().changelist_view(request, extra_context=extra_context)
 
-        # Obtain final queryset from ChangeList object
-        change_list_queryset = response.context_data['cl'].queryset
+        if self.geomap_show_map_on_list is True:
```

Comment:
Perhaps it would be better to write here

```python
if not self.geomap_show_map_on_list:
    return response
```    
and leave the rest unchanged?

In my opinion, the PR will be more concise and clear.

Reply:
Good point.